packages/nimble-components/src/table-column/enum-text/group-header-view/index.ts

```diff
@@ -0,0 +1,62 @@
+import { DesignSystem } from '@microsoft/fast-foundation';
+import { observable } from '@microsoft/fast-element';
+import { styles } from '../../text-base/group-header-view/styles';
+import { template } from './template';
+import type { MappingConfigText } from '../../../mapping/text';
+import type { TableColumnEnumColumnConfig } from '../../enum-base';
+import { TableColumnTextGroupHeaderViewBase } from '../../text-base/group-header-view';
+import type { TableFieldValue } from '../../../table/types';
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-table-column-enum-text-group-header-view': TableColumnEnumTextGroupHeaderView;
+    }
+}
+
+/**
+ * A group header view for enum columns
+ */
+export class TableColumnEnumTextGroupHeaderView extends TableColumnTextGroupHeaderViewBase<
+TableFieldValue,
+TableColumnEnumColumnConfig
+> {
+    /** @internal */
+    public span: HTMLSpanElement | null = null;
+
+    /** @internal */
+    @observable
+    public isValidContentAndHasOverflow = false;
+
+    public getMappingToRender(): MappingConfigText | null {
```

Comment:
Is there a reason why these classes are mixing property getters and `getXYZ()` functions? It seems inconsistent to me, but perhaps I'm missing the distinction of why one is preferred in certain cases.

Reply:
No, it just happened to end up that way. I can make them all properties.